server/lorax_server/models/custom_modeling/flash_mistral_modeling.py

```diff
@@ -342,12 +358,14 @@ def forward(
                 query,
                 torch.select(kv, dim=1, index=0),
                 torch.select(kv, dim=1, index=1),
-                kv_cache[0],
-                kv_cache[1],
+                None if self.kv_dtype == 'fp8' else kv_cache[0],
```

Comment:
I would rather pass in the `kv_cache` values and then have this function decide whether to use them or not (as this seems like a low level implementation detail). One way would be to have a param you pass in here like `is_fp8_kv`.

Reply:
Done.